cluster/gce/gci/flexvolume_node_setup.sh

```diff
@@ -99,31 +99,31 @@ generate_chroot_wrapper() {
     exit 1
   fi
 
-  for driver_dir in ${MOUNTER_PATH}/flexvolume/*; do
+  for driver_dir in "${MOUNTER_PATH}/flexvolume"/*; do
     if [ -d "$driver_dir" ]; then
 
-      filecount=$(ls -1 $driver_dir | wc -l)
-      if [ $filecount -gt 1 ]; then
+      filecount=$(find "$driver_dir" -mindepth 1 -maxdepth 1 -printf "%P\n" | grep -c -v "^\.")
```

Comment:
```
      filecount=$(ls -1 $driver_dir | wc -l)
                  ^---------------^ SC2012: Use find instead of ls to better handle non-alphanumeric filenames.
                        ^---------^ SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
darwin's version of `find` doesn't have `-printf`

the original implementation is way simpler, and since this is used for counting and not the specific paths I think it's acceptable to ignore SC2012 here

Reply:
if we did _actually_ want the values and not the count, we have an established pattern elsewhere:
- use a subshell
  - cd to the target search directory
  - use `find` against `.`
  - trim `./` from the beginning by piping to `cut` or similar

To get filename, use print0 and basename.

+1 on not needing it here

Reply:
Thank you for the comments.
I modified it a little as follows. As far as I run, the result is equivalent to the original way. But, more complicated and not readable.
```
filecount=$(cd "$driver_dir"; find . -mindepth 1 -maxdepth 1 -print0 | xargs -0 -n1 basename | grep -c -v "^\.")
```
Can we ignore SC2012?

Reply:
Specifically for getting a count, that's probably fine.
But generally speaking SC2012 exists because ls output is NOT portable or specified. We should prefer find.

Longer term we should move elaborate logic out of bash.

Reply:
I see. I updated the commit using `-print0` and `basename`. Could you check it?